client/interface/interface.go

```diff
@@ -0,0 +1,34 @@
+package clientinterface
+
+import (
+	"context"
+	"time"
+
+	"github.com/drand/drand/chain"
+)
+
+// Client represents the drand Client interface.
+type Client interface {
+	// Get returns a the randomness at `round` or an error.
+	// Requesting round = 0 will return randomness for the most
+	// recent known round, bounded at a minimum to the `RoundAt(time.Now())`
+	Get(ctx context.Context, round uint64) (Result, error)
+
+	// Watch returns new randomness as it becomes available.
+	Watch(ctx context.Context) <-chan Result
+
+	// Info returns the parameters of the chain this client is connected to.
+	// The public key, when it started, and how frequently it updates.
+	Info(ctx context.Context) (*chain.Info, error)
```

Comment:
`*chain.Info` stands out in that it's the only thing here that is a concrete type...

Reply:
eventually @nikkolasg wants a low-dependency interface layer on chain. not convinced i want to combine that with this